selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/NewWorkspacePageTest.java

```diff
@@ -0,0 +1,519 @@
+          "android-default",
+          "cpp-default",
+          "centos",
+          "centos-go",
+          "nodejs4",
+          "wildfly-swarm",
+          "ceylon-java-javascript-dart-centos",
+          "debian",
+          "debianlsp",
+          "che-in-che",
+          "vert.x",
+          "go-default",
+          "hadoop-default",
+          "java-centos",
+          "java-debian",
+          "java-theia-docker",
+          "java-theia-openshift",
+          "java-centos-mysql",
+          "kotlin-default",
+          "node-default",
+          "openshift-default",
+          "openshift-sql",
+          "php-default",
+          "php-gae",
+          "php5.6-default",
+          "platformio",
+          "python-default",
+          "python-2.7",
+          "python-gae",
+          "rails-default",
+          "selenium",
+          "spring-boot",
+          "tomee-default",
+          "ubuntu",
+          "zend");
+
+  private static final List<String> EXPECTED_MULTI_MACHINE_STACKS =
+      asList(
+          "java-mysql",
+          "java-theia-docker",
+          "java-theia-openshift",
+          "java-centos-mysql",
+          "openshift-sql");
+
+  private static final List<String> EXPECTED_QUICK_START_STACKS_REVERSE_ORDER =
+      asList(
+          "blank-default",
+          "java-default",
+          "dotnet-default",
+          "rails-default",
+          "python-default",
+          "platformio",
+          "php-default",
+          "openshift-sql",
+          "openshift-default",
+          "node-default",
+          "java-theia-openshift",
+          "go-default",
+          "che-in-che",
+          "cpp-default",
+          "android-default");
+
+  private static final List<String> EXPECTED_JAVA_STACKS =
+      asList("java-default", "android-default", "che-in-che", "java-theia-openshift");
+
+  private static final List<String> EXPECTED_FILTERS_SUGGESTIONS =
+      asList("JAVA", "JDK", "JAVA 1.8, THEIA");
+
+  private static final List<String> VALID_NAMES =
+      asList("Wk-sp", "Wk-sp1", "9wk-sp", "5wk-sp0", "Wk19sp", "Wksp-01");
+
+  @Inject private Dashboard dashboard;
+  @Inject private WorkspaceProjects workspaceProjects;
+  @Inject private WorkspaceConfig workspaceConfig;
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private Workspaces workspaces;
+  @Inject private NewWorkspace newWorkspace;
+  @Inject private TestWorkspaceProvider testWorkspaceProvider;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private TestWorkspaceServiceClient testWorkspaceServiceClient;
+  @Inject private SeleniumWebDriverHelper seleniumWebDriverHelper;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  @Inject private DocumentationPage documentationPage;
+  @Inject private WorkspaceOverview workspaceOverview;
+  @Inject private Stacks stacks;
+
+  @BeforeClass
+  public void setup() {
+    dashboard.open();
+  }
+
+  @BeforeMethod
+  public void prepareToTestMethod() {
```

Comment:
It seems we are creating workspace here. If so, lets name the method to explain its intentions, for example: `prepareTestWorkspace()`